i18n/locales/es.ts

```diff
@@ -0,0 +1,376 @@
+      paragraph_three:
+        'En función de los comentarios que recibamos, trabajaremos para lanzar una versión oficial (que llamaremos V1) que incluya 2-3 capítulos completos de lecciones, cuentas y mucho más. Se han planificado 10 capítulos que se irán publicando a medida que se completen. Entre las funciones adicionales se incluyen el aprendizaje colaborativo, las insignias, las misiones secundarias más desafiantes, etc.',
+    },
+
+    contributing: {
+      title: 'Cómo contribuir',
+      paragraph_one:
+        'Esto es un proyecto de código abierto, así que todas las puertas están abiertas para que ayudes a darle forma.',
+
+      feedback: {
+        title: 'Comentarios',
+        paragraph_one:
+          'Una buena forma de contribuir es repasar el capítulo 1 y darnos tu opinión sobre la experiencia general utilizando este formulario.',
+      },
+
+      contribute: {
+        title: 'Contribuir',
+        paragraph_one:
+          'Todas las sugerencias son bienvenidas, incluidos los cambios de contenido, mecánica de juego - realmente cualquier cosa. También estamos siempre buscando ayuda para revisar y <a href="https://leaf-singer-0fc.notion.site/How-to-QA-5177e63f65a4406da01bc57d886b5ac2" target="_blank" rel="noreferrer">Controlar la calidad de los cambios de código</a>. Para mejorar el código, puedes abrir directamente una incidencia o enviar un pull request en GitHub.',
+        paragraph_two:
+          'Si quieres participar en el diseño, la historia o cualquier otra cosa, únete a nosotros en el canal #saving-satoshi en el slack <a href="https://bitcoin.design/contribute/" target="_blank" rel="noreferrer">Bitcoin Design</a>. Menciona cómo te gustaría contribuir y te ayudaremos a orientarte en la dirección correcta.',
+      },
+    },
+
+    contributors: {
+      title: 'Disfrutalo!',
+      contributions_by: 'Contribuciones de',
+      many_more:
+        ', <href="https://VEINTIUNO.world/libro/salvando-satoshi/" target="_blank">@VEINTIUNO.world</a> y <a href="https://github.com/saving-satoshi/saving-satoshi/graphs/contributors" target="_blank" rel="noreferrer">muchos más</a>.',
+    },
+
+    satoshi_needs_you: 'Ahora date prisa, Satoshi te necesita.',
+  },
+
+  ///CHAPTERS TITLE PAGE
+  chapter_one: {
+    title: 'Secretos a la vista de todos',
+    paragraph_one:
+      'El año es 2139. Faltan dos semanas para que se extraiga el último bitcoin. Desde hace meses, un reloj hace tictac en la Plaza Satoshi. El mundo espera el último bloque. De repente, la red se detiene.',
+
+    ///CHAPTER 1
+    intro_one: {
+      title: 'Génesis',
+      paragraph_one: 'Unos instantes después, tu Pantalla se activa.',
+      paragraph_two: '-Deborah Chunk: "Thomas Vanderpoole. Como CEO de BitRey, diriges uno de los mayores pools de minería de bitcoins del mundo. También fabrica máquinas de minería bitcoin. ¿Qué está ocurriendo? ¿Está muriendo bitcoin?”',
```

Comment:
Bitcóin - bitcoines ([reference from Fundeú](https://www.fundeu.es/recomendacion/bitcoin-bitcoins-bitcoines/)). This one also applies to other places, but I won't spam the PR highlighting them all.

```suggestion
      paragraph_two: '-Deborah Chunk: "Thomas Vanderpoole. Como CEO de BitRey, diriges uno de los mayores pools de minería de bitcoines del mundo. También fabricas máquinas para minar bitcóin. ¿Qué está ocurriendo? ¿Está muriendo bitcoin?”',
```